x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -78,13 +78,83 @@ export function uniquePidForProcess(passedEvent: ResolverEvent): string {
   }
 }
 
+/**
+ * Returns the pid for the process on the host
+ */
+export function processPid(passedEvent: ResolverEvent): string {
+  if (event.isLegacyEvent(passedEvent)) {
+    return String(passedEvent.endgame.pid);
+  } else {
+    return String(passedEvent.process.pid);
+  }
+}
+
 /**
  * Returns the process event's parent pid
  */
 export function uniqueParentPidForProcess(passedEvent: ResolverEvent): string | undefined {
   if (event.isLegacyEvent(passedEvent)) {
-    return String(passedEvent.endgame.unique_ppid);
+    return String(passedEvent.endgame.ppid);
```

Comment:
why this change?

Reply:
> this should be unique_pid

@oatkiller Just checking - you mean `unique_ppid` here, right?